mernarezk/assignment2/BinaryTree.java

```diff
@@ -0,0 +1,57 @@
+package Assignment2;
+
+/**
+ * Simple Binary Tree class that takes int values since BOTH problems are
+ * defined by taking in values and for testing simplicity methods params in
+ * assignments take direct int's rather than node references.

```

Comment:
This is fine. :) Very good that you added a comment about it. During an interview it's really important to say these things to the interviewer - they will usually agree to proceed, unless it's something that overly simplifies the algorithm. In this case, I don't think anyone would have any objections.

Reply:
^_^ thanks!